src/lib/replication/service/repl_service_impl.cpp

```diff
@@ -46,6 +47,14 @@ void ReplicationServiceImpl::stop() {
     m_rd_map.clear();
 }
 
+hs_stats ReplicationServiceImpl::get_cap_stats() const {
```

Comment:
we should be able to get total/used space of a particular repl_dev(a.k.a PG).
As we limit one PG to one pdev in HO and that information is only available in there,  we may better handle the information there?

Reply:
We thought about doing it in PG level, but current nuobject proto buffer definition only need total/used per SM. 

We can explore the needs for per PG level stats if there is a use case.